routes/controllers/snippets.controllers.js

```diff
@@ -184,9 +184,73 @@ const createSnippet = async (req, res, next) => {
   }
 };
 
+const createComment = async (req, res, next) => {
+  const { userId, snippetId, comment } = req.body;
+  const { auth: token } = req.cookies;
+
+  const decodedId = jwt.decode(token);
+
+  try {
+    const inValidUser = String(userId) !== String(decodedId);
+
+    if (inValidUser) {
+      throw createError(403, NO_AUTHORITY_TO_ACCESS);
+    }
+
+    await Snippet.findByIdAndUpdate(
+      snippetId,
+      { $push: { commentList: { creator: userId, content: comment, createdAt: Date.now() } } },
+      { new: true }
+    );
+
+    res.send({ result: OK });
+  } catch (error) {
+    if (error.status) {
+      next(error);
+
+      return;
+    }
+
+    next({ message: UNEXPECTED_ERROR });
+  }
+};
+
+const deleteComment = async (req, res, next) => {
+  const { userId, commentId, snippetId } = req.body;
+  const { auth: token } = req.cookies;
+
+  const decodedId = jwt.decode(token);
+
+  try {
+    const inValidUser = String(userId) !== String(decodedId);
+
+    if (inValidUser) {
+      throw createError(403, NO_AUTHORITY_TO_ACCESS);
+    }
+
+    await Snippet.findByIdAndUpdate(
+      snippetId,
+      { $pull: { commentList: { _id: commentId } } },
+      { new: true }
+    );
+
+    res.send({ result: OK });
```

Comment:
마찬가지로 status code 추가 부탁드립니다!

```suggestion
    res
      .status(200)
      .send({ result: OK });
```